geopandas/geodataframe.py

```diff
@@ -1401,18 +1401,29 @@ def merge_geometries(block):
         return aggregated
 
     # overrides the pandas native explode method to break up features geometrically
-    def explode(self, column=None, **kwargs):
+    def explode(self, column=None, ignore_index=False, add_multiindex=True):
         """
         Explode muti-part geometries into multiple single geometries.
 
         Each row containing a multi-part geometry will be split into
         multiple rows with single geometries, thereby increasing the vertical
         size of the GeoDataFrame.
 
-        The index of the input geodataframe is no longer unique and is
-        replaced with a multi-index (original index with additional level
-        indicating the multiple geometries: a new zero-based index for each
-        single part geometry per multi-part geometry).
+
+        Parameters
+        ----------
+        column : string, default None
+            If None, MultiGeometries in geometry column are converted to Single ones.
+            Else, explode another column following pandas behaviour.
+        ignore_index : bool, default False
+            If true, reset index to 0..n-1. Pandas > 1.1.0 needed for this argument.
```

Comment:
```suggestion
           If True, the resulting index will be labelled 0, 1, …, n - 1.
```
Let's follow the pandas docstring.


Add a separate note below:

```rst
.. note:: ignore_index requires pandas 1.1.0 or newer.
```